src/main/java/com/trueffect/sql/crud/PersonCrud.java

```diff
@@ -40,26 +42,33 @@ public static Person insertrenterUser(int idJob, Person renterUser) throws Excep
                 " VALUES ('"+typeIdentifier+"','"+identifier+"','"+lastName+"', '"
                         +firstName+"','"+ genre+"','"+birthay+"','2017-12-08','"+idJob+"',null,null,'Active')";
       
-                consulta.execute(sql);
+                query.execute(sql);
                 String msg = "The user " + firstName + " " + lastName + " has been successfully inserted";
                 renterUserInserted = getByTypeIdentifier(typeIdentifier, identifier);
               } catch (Exception e) {
                 throw new ErrorResponse(CodeStatus.INTERNAL_SERVER_ERROR, e.getMessage());
-            }
-            PostgresSQLConnection.closeDB();
+            }finally{
+                 try {
```

Comment:
You have methods for close and create conection, can you create a method for close  sql and put in the respective class